site/code-snippets/web5/quickstart.js

```diff
@@ -34,7 +34,7 @@ export async function updateTextRecord(record) {
 export async function deleteTextRecord(web5, did, recordId) {
   const deleteResult = await web5.dwn.records.delete({
     message: {
-      recordId,
+      recordId: record.id,
```

Comment:
```suggestion
      recordId: recordId
```

It needs to use that argument from line 34